src/components/paragraph/button/index.tsx

```diff
@@ -0,0 +1,29 @@
+import { v4 as uuidv4 } from 'uuid'
+import Link from 'next/link'
+
+function isRequestValid(paragraph: any) {
+  return (
+    paragraph.field_button_label !== null &&
+    paragraph.field_button_link !== null &&
+    paragraph.field_button_link?.uri !== null
+  )
+}
+
+const Button = ({ paragraph }): JSX.Element => {
+  if (!paragraph || !isRequestValid(paragraph)) return
+
+  const {
+    field_button_label: fieldButtonLabel,
```

Comment:
I don't object to this pattern, but, I also don't know if it's necessary.

Would it be possible create a method that would destructure an object into consts from its properties like this and convert them to camelCase at the same time? i.e. something 'magic'. I don't think we need it for this ticket, but I wonder how possible that would be.

Reply:
Removed